include/enums/cpp_metadata_type.h

```diff
@@ -47,26 +48,26 @@ enum class MetaDataType {
 //! Helper method to convert between MetaDataType and CMetaDataType
 inline CMetaDataType convert_metadata_type(MetaDataType type) {
     CMetaDataType t;
-    switch(type) {
-        case MetaDataType::dbl :
+    switch (type) {
```

Comment:
We should probably add the ``invalid`` type to these switch cases too.

Reply:
done